Azure Synapse fixes (#632)
rudderlabs/rudderstack-docs: docs/data-warehouse-integrations/azure-synapse.mdx

Find the open-source transformer code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/azure_synapse">GitHub repository</a>.
</div>

> ## Prerequisites

Author: ```suggestion
## Pre-requisites
```

> ## Pre-requisites

Author: ```suggestion
## Prerequisites
```

> Before setting up Azure Synapse as a destination in RudderStack, make sure you have an [Azure subscription](https://azure.microsoft.com/en-in/get-started/synapse-analytics/) for the SQL Data Warehouse. 

Reviewer: The link is here is pointed to the wrong place, can we change this to: "Before setting up Azure Synapse as a destination in RudderStack, make sure you have an Azure subscription and that the [Azure Synapse Workspace] (https://docs.microsoft.com/en-in/azure/synapse-analytics/quickstart-create-workspace) is created "

> Before setting up Azure Synapse as a destination in RudderStack, make sure you have an [Azure subscription](https://azure.microsoft.com/en-in/get-started/synapse-analytics/) for the SQL Data Warehouse. 

Author: ```suggestion
Before setting up Azure Synapse as a destination in RudderStack, make sure you have an Azure subscription and created the [Azure Synapse Workspace](https://docs.microsoft.com/en-in/azure/synapse-analytics/quickstart-create-workspace).
```

> Additionally, you need a dedicated SQL pool created in Azure Synapse. If you do not have one, you can create it by following the [Azure Synapse documentation](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/create-data-warehouse-portal).

Reviewer 2: ```suggestion
Additionally, you need to create a dedicated SQL pool in Azure Synapse. You can create it by following the [Azure Synapse documentation](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/create-data-warehouse-portal).
```
